### PR TITLE
Made minor changes to nats doc

### PR DESCRIPTION
**Pull request by blackfly19**
Made minor changes and tested the working of nats-streaming connector according to the documentation.

### CODE REVIEW

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 1]
```diff
@@ -13,30 +13,29 @@ You will also need NATS Streaming server setup which is reachable from the Fissi
...
 ```
+
 {{% /notice %}}
 
 ```sh
 $ kubectl apply -f nats-dep.yaml
 NAME                                         READY   STATUS    RESTARTS   AGE
 nats-streaming-deployment-646768fcfd-qtpmk   1/1     Running   0          8s
 ```
-You can find above file reference [here](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/nats-streaming-server/nats-dep.yaml)
```
Comment (girishg4t) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:34]: This references we need to give, don't remove that

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 2]
```diff
@@ -60,34 +59,33 @@ $ curl nats.default.svc.cluster.local:8222
...
-
+4. Fission NATS streaming trigger takes the response of consumer function (consumer) and drops the message in a response queue named `response`. If there is an error, the message is dropped in error queue named `error`.
 
 ## Building the app
 
 ### Producer Function
 
 The producer function is a go program which creates a message and drops into a NATS streaming queue `request`.
 For brevity all values have been hard coded in the code itself.
-There are different ways of loading this function into cluster, i have tried by creating the deployment. The docker and other file is present under nats-streaming-http-connector/test/producer folder [here](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/producer).
```
Comment (girishg4t) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:81]: same here

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 3]
```diff
@@ -60,34 +59,33 @@ $ curl nats.default.svc.cluster.local:8222
...
 
-Below are the steps i did
 ```sh
-$ docker build . -t producer:latest 
-$ kind load docker-image producer:latest --name kind-1 
+$ docker build . -t producer:latest
+$ kind load docker-image producer:latest --name kind
 $ kubectl apply -f deployment.yaml //replicas is set to 0 when deployed
 ```
-[Go file](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/producer/main.go)
```
Comment (girishg4t) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:89]: same

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 4]
```diff
@@ -112,20 +110,20 @@ func main() {
 		sc.Publish("hello", []byte("Test"+strconv.Itoa(i)))
 	}
 	fmt.Println("Published all the messages")
-	
+
 	select {}
 }
 ```
-[Dockerfile](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/producer/Dockerfile)
```
Comment (girishg4t) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:119]: same

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 5]
```diff
@@ -134,13 +132,13 @@ RUN go mod download
...
 RUN go build -a -o /go/bin/main
-FROM alpine:3.12 as base
+FROM alpine:3.14 as base
 RUN apk add --update ca-certificates
 COPY --from=builder /go/bin/main /
 
-ENTRYPOINT ["/main"] 
+ENTRYPOINT ["/main"]
 ```
-[deployment.yaml](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/producer/deployment.yaml)
```
Comment (girishg4t) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:143]: same

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 6]
```diff
@@ -159,21 +157,23 @@ spec:
...
+$ kubectl get deployment nats-pub
 NAME       READY   UP-TO-DATE   AVAILABLE   AGE
-nats-pub   0/0     0            0           
+nats-pub   0/0     0            0
 
 ```
+
 ### Consumer function
 
-The consumer function is golang function which takes the body of the request, appends a "Hello" and returns the resulting string, the code is present in nats-streaming-http-connector/test/consumer [here](https://github.com/fission/keda-connectors/tree/master/nats-streaming-http-connector/test/consumer)
```
Comment (girishg4t) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:176]: this will also be needed

**Review by girishg4t — CHANGES_REQUESTED** [review]
You can check the earlier review comments, the earlier review has asked me to add all the references we should not remove that

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md (hunk 1 above)
Comment (blackfly19) [reply docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:34]: This reference I have added in this line
You can also setup NATS streaming server with this yaml file.(Monitoring is already configured).
Should I still add here?

**Review by girishg4t — APPROVED** [review]
LGTM

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**